cmd/tink-worker/internal/registry_test.go

```diff
@@ -0,0 +1,72 @@
+package internal
+
+import (
+	"context"
+	"errors"
+	"io"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/packethost/pkg/log"
+	"github.com/stretchr/testify/assert"
+)
+
+func setupTestLogger() log.Logger {
+	service := "github.com/tinkerbell/tink"
+	logger, err := log.Init(service)
+	if err != nil {
+		panic(err)
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#dont-panic

I'd use `t.Fatalf` instead and pass `*testing.T` as a parameter to this function.

Reply:
done